hkmc2/shared/src/test/mlscript/ctx/Summon.mls

```diff
@@ -13,11 +13,16 @@ use[Str]
 //│ = "42"
 
 :e
+:re
 use[Num]
-//│ ╔══[ERROR] Cannot query instance for use-expression of type Num
-//│ ║  l.16: 	use[Num]
-//│ ║        	    ^^^
-//│ ╙── Missing instance: Expected: Num; Available: Int, Str
+//│ ╔══[ERROR] Cannot query instance of type Num (type variable T) for call: 
+//│ ║  l.17: 	use[Num]
+//│ ║        	^^^^^^^
+//│ ╟── Required by contextual parameter declaration: 
+//│ ║  l.98: 	fun use[T](using instance: T) = instance
+//│ ║        	                 ^^^^^^^^^^^
+//│ ╙── Missing instance: Expected: Num (type variable T); Available: Int, Str
+//│ ═══[RUNTIME ERROR] Error: MLscript call unexpectedly returned `undefined`, the forbidden value.
```

Comment:
Now it generates a runtime error (used to be a no-op if it cannot find the instance).